EasyPostTest/ShipmentTest.cs

```diff
@@ -310,5 +310,39 @@ public class ShipmentTest {
             ShipmentList nextShipmentList = shipmentList.Next();
             Assert.AreNotEqual(shipmentList.shipments[0].id, nextShipmentList.shipments[0].id);
         }
+
+        //Smart rate
+        [TestMethod]
+        public void ApiCallIsNotNull()
+        {
+            Shipment shipment = Shipment.Create(parameters);
+            List<Smartrate> smartRateResult = shipment.GetSmartrates();
+            Assert.IsNotNull(smartRateResult);
+        }
+
+        [TestMethod]
+        public void TestGetSmartrates()
+        {
+            Shipment shipment = Shipment.Create(parameters);
+            List<Smartrate> smartRateResult = shipment.GetSmartrates();
```

Comment:
`smartRateResult` should probably be `smartrateResult` So that "smartrate" is either capitalized or lowercased whenever used depending so that it feels like a single word